packager/react-packager/src/Resolver/polyfills/Array.prototype.es6.js

```diff
@@ -5,54 +5,51 @@
  * @polyfill
```

Comment:
i.e. it should be fine given the changes in `packager/react-packager/src/Resolver/index.js`